__tests__/index.test.js

```diff
@@ -36,3 +36,43 @@ describe("GET /api/categories", () => {
       });
   });
 });
+
+describe("GET /api/reviews/:review_id", () => {
+  test("returns a review object with the specified id and correct properties ", () => {
+    return request(app)
+      .get("/api/reviews/3")
+      .expect(200)
+      .then(({ body }) => {
+        const { review } = body;
+        expect(review).toEqual(
+          expect.objectContaining({
+            review_id: expect.any(Number),
```

Comment:
Because this is based on a singular, static review - it would be sufficient (and better) here to explicitly say the object you are receiving back! You will be able to find this in the test-data files.